internal/importer/importer.go

```diff
@@ -88,217 +102,216 @@ func (i azureImporter) InitWritesProblem(token wharfapi.Token, provider wharfapi
 	return true
 }
 
-func (i azureImporter) ImportProjectInGroupWritesProblem(groupName, projectName string) bool {
+func (i *azureImporter) ImportRepositoryWritesProblem(orgName, projectNameOrID, repoNameOrID string) bool {
 	http.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
```

Comment:
Is this ok because it uses dockerinternals network?

Reply:
No. It's not OK. But I didn't want to tackle that one as well in this PR.

But since the addition of `httputils.NewClientWithCerts` in https://github.com/iver-wharf/wharf-provider-azuredevops/pull/23/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R48-R55 we should remove this completely, or as https://github.com/iver-wharf/wharf-provider-azuredevops/issues/10 suggests, make it opt-in.